docs/virtual_machines/persistent_tpm_and_uefi_state.md

```diff
@@ -0,0 +1,85 @@
+# Persistent TPM and UEFI state
+
+**FEATURE STATE:** KubeVirt v1.0.0
+
+For both TPM and UEFI, libvirt supports persisting data created by a virtual machine as files on the virtualization host.  
+In KubeVirt, the virtualization host is the virt-launcher pod, which is ephemeral (created on VM start and destroyed on VM stop).  
+As of v1.0.0, KubeVirt supports using a PVC to persist those files. KubeVirt usually refers to that storage area as "backend storage".
+
+## Backend storage
+
+KubeVirt automatically creates backend storage PVCs for VMs that need it. However, the admin must first enable the `VMPersistentState` feature gate, and tell KubeVirt which storage class to use by setting the `vmStateStorageClass` configuration parameter in the KubeVirt Custom Resource (CR).  
+The storage class must support read-write-many (RWX) in filesystem mode (FS).  
+Here's an example of KubeVirt CR that sets both:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    vmStateStorageClass: "nfs-csi"
+    developerConfiguration:
+      featureGates:
+      - VMPersistentState
+```
+
+### Limitations
+
+- As mentioned above, the backend storage PVC can only be created using a storage class that supports RWX FS. There is ongoing work to support block storage in future versions of KubeVirt.
+- Backend storage is currently incompatible with VM snapshot. It is planned to add snapshot support in the future.
+
+## TPM with persistent state
+
+Since KubeVirt v0.53.0, a TPM device can be added to a VM (with just `tpm: {}`). However, the data stored in it does not persist across reboots.  
+Support for persistence was added in v1.0.0 using a simple `persistent` boolean parameter that default to false, to preserve previous behavior.  
+Of course, backend storage must first be configured before adding a persistent TPM to a VM. See above.  
+Here's a portion of a VM definition that includes a persistent TPM:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vm
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          tpm:
+            persistent: true
+```
+
+### Uses
+- The Microsoft Windows 11 installer requires the presence of a TPM device, even though it doesn't use this. Persistence is not required in this case however.
```

Comment:
what do you mean by "even though it doesn't use this"?

Reply:
The installer forces you to have a TPM chip for no good reason. Neither it nor a freshly installed Windows will actually make use of the TPM (as long as you don't enable bitlocker).
So if you don't plan on using bitlocker, just add a non-persistent TPM to your VM and you'll be all set!
Do you think the docs need more info on this?

Reply:
That's..... interesting? :D

Alright then, good to know!